src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -90,7 +106,7 @@ public PersistentDataStore() {
     List<Conversation> conversations = new ArrayList<>();
 
     // Retrieve all conversations from the datastore.
-    Query query = new Query("chat-conversations");
+    Query query = new Query("chat-conversations").addSort("creation_time");
```

Comment:
Is there a reason for sorting by creation time here? (can we add a comment to indicate why?)